app/components/state_diagram_component.html.erb

```diff
@@ -0,0 +1,13 @@
+<details class="govuk-details" data-module="govuk-details">
+  <summary class="govuk-details__summary">
+    <span class="govuk-details__summary-text">
+      View diagram of application flow
+      <span class="govuk-visually-hidden">(accessible example presented after diagram)</span>
```

Comment:
This is... quite classy!?

Reply:
GDS sure like their classes!

Reply:
Once we get the GOVUK components gem implemented, this will all be hidden away 😉

Reply:
No no — I meant the text! 🤵

Reply:
Ooh, bit like “your feature presentation will start after these messages“ 😂